mantidproject/mantid
# ISIS Powder Gem texture mode fingerprint

**Description of work.**
Added `texture_mode` state to vanadium spline fingerprint for ISIS Powder GEM.

**To test:**
Copy the GEM_test folder from `\\olympic\babylon5\Public\Sam Jenkins` to your computer, then edit the `Gem_config_example.yaml` so that the paths within are complete. then run the following script.
```
from isis_powder import Gem

config_file_path = r"C:\Users\axj71237\Documents\Isis powder\Gem\Gem_config_example.yaml"
Gem_example = Gem(config_file=config_file_path,user_name="test",texture_mode=True)
Gem_example.create_vanadium(first_cycle_run_no="85105")
Gem_example.focus(input_mode="Individual",
                  mode="Rietveld", run_number="85105",
                  vanadium_normalisation=True)
```
Then check that the `VanSplined` file generated in calibration/cycle_18_1/ contain `texture_mode` in their name. then try again with `texture_mode=False` it should generate a second `VanSplined` file without `texture_mode` in the name.

Fixes #23197

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
